src/command.ts

```diff
@@ -4,30 +4,32 @@ import * as pty from './nodePty'
 import { PathLike } from 'fs'
 import { execFileSync } from 'child_process'
 
+export interface CommandOptions {
+    cwd?: string
+    forceSandboxed?: boolean
```

Comment:
Just sandboxed is enough I guess.

Reply:
I think it better communicate that it overrides the IS_SANDBOXED with `force` prefix

Reply:
Right but that is something no one should use in the code unless it is for tests, so we could document that. Or even rename it to forceSandbox instead if you prefer

Reply:
Could also be useful when debugging. Renamed it to `forceSandbox`.

We could also reverse and rename it to `alwaysRunInHost`. This would be helpful when running in host doesn't make sense. But won't do it now I guess, until we have a use case.